code/modules/mob/living/life.dm

```diff
@@ -1,7 +1,8 @@
 /mob/living/Life(seconds, times_fired)
 	set invisibility = 0
 	set background = BACKGROUND_ENABLED
-
+	if(istype(loc, /obj/machinery/sleeper))
```

Comment:
This won't work, as children override Life completely without calling `..()`.

Also don't like the idea of checking with istype for a sleeper every life tick. I would suggest making a "stasis" status_effect.

Reply:
Fucking

Reply:
I considered status effects but I dont want to deal with years of "halp status effect didn't disable properly"

Reply:
I'll try to come up with something.

Reply:
@ninjanomnom this code is not ready to go, look at this shit.

Reply:
well oranges it's either this or I add a var to every object to count as stasis/a component

Reply:
A component would make sense, wouldn't it?

Reply:
Probably but the same issue with status effect

Reply:
it's acceptable for a pr meant to just balance prety much @optimumtact 

Though I guess yeah since it's named rework it should be better made

Reply:
Balance is never an excuse for bad code sorry